Fix ipager to compile with GCC 4.3.

Patch taken from Gentoo bug #173562.

X-Gentoo-Bug: 173562
Signed-off-by: Robin H. Johnson <robbat2@gentoo.org>

--- a/iconfig.cpp	2008-08-22 21:35:16.000000000 +0600
+++ b/iconfig.cpp	2008-08-22 21:35:20.000000000 +0600
@@ -30,7 +30,7 @@
 #include <iostream>
 #include <fstream>
 #include <sstream>
-
+#include <stdlib.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 
--- a/wm.cpp	2008-08-22 21:52:36.000000000 +0600
+++ b/wm.cpp	2008-08-22 21:52:40.000000000 +0600
@@ -27,7 +27,7 @@
 
 #include <time.h>
 #include "atoms.h"
-
+#include <stdlib.h>
 
 WM * WM::m_instance = 0;
 bool WM::x_error = false;
